README.md

```diff
@@ -45,7 +45,7 @@ In order to learn more, pleasee see [purpose of Sumo Logic Distribution for Open
 ## Supported OS and architectures
 
 | Linux                         | MacOS                         | Windows                     |
-| ----------------------------- | ----------------------------- | --------------------------- |
+|-------------------------------|-------------------------------|-----------------------------|
```

Comment:
Looks like we're using different Markdown (table) formatters :sweat_smile: What is the formatter you use locally?